### DISCUSSION TITLE
Different color result between ImageMagick 6.5.4-7 and ImageMagick 6.9.12-50

### DISCUSSION THREAD

**Discussion by gzhcoder** [ImageMagick/ImageMagick6#189] [General] [answered]
I have had a PHP Website working for 7 years and have had no problems
until a few weeks ago I replaced my ImageMagick with a newer one.

ImageMagick (old): ImageMagick 6.5.4-7
OS (old): CentOS release 6.4 (Final)

ImageMagick (new): ImageMagick 6.9.12-50
OS (new): Red Hat Enterprise Linux release 8.5 (Ootpa)

I run following command in ImageMagick 6.5.4-7 and ImageMagick 6.9.12-50, it returns different color results.

[root@ImageMagickTest ~]# /usr/bin/convert -colorspace RGB "/home/share/from_file.jpeg" "/home/share/to_file.jpeg"

JPG（to_file.jpeg） file generated with ImageMagick 6.9.12-50 looks darker in color.
The JPG（to_file.jpeg） file of the new version is not as clear as the old version.
When I use the file_get_contents() to read the contents of to_file.jpg into a string.
The contents of the string are also different.

What is the reason for this and is there any easy way to maintain compatibility?
How to set the new version to get the same results as the old version?

Steps to Reproduce
/usr/bin/convert -colorspace RGB "/home/share/from_file.jpeg" "/home/share/to_file.jpeg"

Images
from_file.jpeg
![from_file](https://user-images.githubusercontent.com/23540382/177265957-f80582c9-d656-4c86-a7db-a2270b1b473a.jpeg)

to_file.jpeg
![to_file](https://user-images.githubusercontent.com/23540382/177265961-693b9ea9-5ada-4d6f-8c98-594315a05cd7.jpeg)

**Comment by urban-warrior** [comment, marked as answer]
Try `-colorspace sRGB`.  RGB is linear, sRGB, non-linear.

**Reply by gzhcoder** [reply]
Dear urban-warrior,
Thank you very much for your reply.
After using the `-colorspace sRGB` the picture looks like the original picture.
Although the contents of the picture is still inconsistent, but the effect is acceptable to my customers.
